### PR TITLE
Update pmu_armv8.h

### PR DESCRIPTION
**Pull request by EdmundPlayer**
Added Armv8.1-M PMU macro definitions and functions.

### CODE REVIEW

File: CMSIS/Core/Include/pmu_armv8.h [hunk 1]
```diff
@@ -31,6 +31,1036 @@
 #ifndef ARM_PMU_ARMV8_H
 #define ARM_PMU_ARMV8_H
 
+/** \brief PMU Events */
 
-#endif /* ARM_PMU_ARMV8_H */
+#define ARM_PMU_SW_INCR                              0x0000
```
Comment (JonatanAntoni) [inline CMSIS/Core/Include/pmu_armv8.h]: We should add descriptive documentation to all these events. What does this event "count"?

**Comment by JonatanAntoni** [comment]
@GuentherMartin, the typedef is in #833.

File: CMSIS/Core/Include/pmu_armv8.h (hunk 1 above)
Comment (EdmundPlayer) [reply CMSIS/Core/Include/pmu_armv8.h]: Done.

**Comment by EdmundPlayer** [comment]
Created new pull request #865 for new PMU changes. Closing this original pull request.

**Pull request closed without merging**